LibreHardwareMonitor/UI/TypeNode.cs

```diff
@@ -95,6 +95,10 @@ public TypeNode(SensorType sensorType, Identifier parentId, PersistentSettings s
                 Image = Utilities.EmbeddedResources.GetImage("loudspeaker.png");
                 Text = "Noise Levels";
                 break;
+            case SensorType.Humidity:
+                Image = Utilities.EmbeddedResources.GetImage("humidity.png");
```

Comment:
What's the source of this icon? Is it royalty free?

Reply:
"flow.png" is the source :)